e2e/app/aou-elements/clr-icon-link.ts

```diff
@@ -1,23 +1,24 @@
 import {Page, WaitForSelectorOptions} from 'puppeteer';
 import BaseElement from './base-element';
+import TextOptions from './text-options';
 import {findIcon} from './xpath-finder';
 
 export default class ClrIconLink extends BaseElement {
    
   static async forLabel(
      page: Page,
-     aElementName: string,
+     textOptions: TextOptions,
      shape: string,
      waitOptions: WaitForSelectorOptions = {visible: true},
      throwErr = true): Promise<ClrIconLink> {
 
     let element: ClrIconLink;
     try {
-      const iconElement = await findIcon(page, aElementName, shape, waitOptions);
+      const iconElement = await findIcon(page, textOptions, shape, waitOptions);
```

Comment:
Do you need to catch and rethrow, or will the framework & language do most of this for you?

Reply:
`jest` will throws exception too. I'm printing out `textOptions` for manual checking if fail